src/Agent/NewRelic.Api.Agent/NewRelic.Api.Agent.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
Did we determine if this is necessary?

Reply:
It's not necessary. The API reference is used for some shared WCF service code, so I reverted that change and updated the shared project to conditionally reference the API project.